platform/core/tests/test_sveipevern.py

```
@@ -0,0 +1,394 @@
+    Vernet skal ikke bli et driftsavbrudd: den første raden noen hadde
+    lukket ville avbrutt transaksjonen for alle de andre.
+
+    MUTASJONEN SOM DREPER DENNE: bytt den stille rettingen mot en
+    RAISE EXCEPTION.
+    """
+    tekst = MIGRASJON.read_text(encoding="utf-8")
+    kropp = re.search(
+        r"CREATE FUNCTION sveipefunn_lukkevern\(\).*?\nEND \$\$;",
+        tekst, re.S)
+    assert kropp, "vaktfunksjonen finnes ikke"
+    kode = "\n".join(l for l in kropp.group(0).splitlines()
```

Ruff (0.16.3)

[error] Ambiguous variable name: `l`

(E741)